docs/source/user/supporting-elements.rst

```diff
@@ -1,20 +1,146 @@
+The bounds argument to the constructor provides L and U, and since TIA (atomic input type) is not passed,
+TIA is inferred from the type of the public bounds.
+The output domain is simply `AllDomain<i32>`, or "the set of all 32-bit signed integers."
+
 
 Metrics and Measures
 --------------------
+A metric is a function that computes the distance between two elements of a set.
+In OpenDP, a concrete example is the type `AbsoluteDistance<u8>`,
+read as "the absolute distance metric where distances are expressed in unsigned 8-bit integers."
+Another example is `SymmetricDistance`, or "the symmetric distance metric",
+which is used to count the number of additions or removals to convert one dataset into another dataset.
 
-Distances
----------
+Some common metrics are:
+
+* SymmetricDistance, computed as the number of additions or removals to convert one dataset into another dataset
+* SubstituteDistance, computed as the number of substitutions to convert one dataset into another dataset
+* AbsoluteDistance<T>, computed as the absolute difference between any two numeric scalars of type T
+* L1Distance<T>, computed as the L1 distance between any two numeric vectors of type T
+* L2Distance<T>, computed as the L2 distance between any two numeric vectors of type T
+
+A measure is a function for computing a number from a subset of a set.
+In OpenDP, a concrete example is the type `MaxDivergence<f64>`,
+read as "the max divergence metric where numbers are expressed in terms of 64-bit floats."
+The max divergence metric has distances that correspond exactly to epsilon in the pure definition of differential privacy.
+
+Some common measures are:
+
+* MaxDivergence<T>, which corresponds to pure-DP, where epsilon has type T
+* SmoothedMaxDivergence<T>, which corresponds to approximate-DP, where epsilon and delta have type T
+
+Consider that it is never necessary to actually evaluate these functions!
+The sole purpose of metrics and measures is to provide the units for distances.
 
 Relations
 ---------
+We assert the privacy properties of a Transformation or Measurement's function via a relation.
+The relation accepts an input distance `d_in` and an output distance `d_out`,
+where `d_in` and `d_out` are distances in terms of the input and output metric or measure.
+
+Let `d_X` be the input metric, `d_Y` be the output metric or measure,
+and `f` be the function in the Transformation or Measurement.
+
+The relation fails if there exists two members of the input domain, x and x', that satisfy two conditions:
+
+#. d_X(x, x') <= d_in. The inputs are d_in-close.
+#. d_Y(f(x), f(x')) > d_out. But the outputs aren't d_out-close!
+
+In plaintext:
+There should be no perturbation to the input (by at most distance `d_in`) that can significantly influence the output (greater than distance `d_out`).
+
+The typical perturbation is the adjustment made to a dataset by adding or removing a user.
+
+
+Distances
```

Comment:
I'd rename this section something like, Putting It Together. Also, a little more explanation of why we're doing all this would help cement the concepts.

(Also have a section on Distances, as a concept, above after Metrics and Measures.)